pkg/kubelet/kubelet.go

```diff
@@ -312,8 +314,17 @@ func makePodSourceConfig(kubeCfg *kubeletconfiginternal.KubeletConfiguration, ku
 		}
 	}
 
+	if updatechannel == nil {
+		for i, client := range kubeDeps.KubeTPClients {
+			source := kubetypes.ApiserverSource + strconv.Itoa(i)
+			klog.Infof("Watching apiserver %s", source)
+			uc := cfg.Channel(source)
+			config.NewSourceApiserver(client, nodeName, uc)
+		}
+	}
+
 	if kubeDeps.KubeClient != nil {
-		klog.Infof("Watching apiserver")
+		klog.Info("Watching apiserver")
```

Comment:
Just curious, why change Infof to Info?

Reply:
func Infof(format string, args ...interface{})
this my understanding is to take a format with parameters whereas Info does not.